tests/sortition_test.cpp

```diff
@@ -0,0 +1,399 @@
+#include <gtest/gtest.h>
+
+#include "common/types.hpp"
+#include "util_test/util.hpp"
+
+namespace taraxa::core_tests {
+
+struct SortitionTest : BaseTest {
+  std::vector<FullNodeConfig> node_cfgs = make_node_cfgs(1);
+};
+
+vec_trx_t generateTrxHashes(size_t count) {
+  vec_trx_t res;
+  for (size_t i = 0; i < count; ++i) {
+    res.push_back(dev::h256(i));
+  }
+  return res;
+}
+
+SyncBlock createBlock(uint64_t period, uint16_t efficiency, size_t dag_blocks_count = 5) {
+  // produced transactions count should be equal to or multiple of this value to produce block with accurate enough
+  // efficiency
+  const size_t TRX_COUNT = 100 * ONE_PERCENT;
```

Comment:
I think for both `TRX_COUNT` and `ONE_PERCENT` we should use google-style -> `kTrxCount` and `kOnePercent`